src/components/applications/ApplicationTimeline.js

```diff
@@ -22,6 +21,7 @@ const useStyles = makeStyles((theme) => ({
 export const ApplicationTimeline = ({stages, currentStageId, status}) => {
 
     const classes = useStyles()
+    const theme = useTheme()
 
     let isBeforeCurrent = true
```

Comment:
Po co nam lety jak mamy useStaty (pewnie pytanie wynika z mojego braku wiedzy)

Reply:
useState przerenderuje cały komponent, więc nie wiem, czy jest to konieczne, zwłaszcza że to ma być tylko taka wewnętrzna flaga